[FEATURE] Traduction ES & NL : mire de connexion & inscription (PIX-15189)

pix-service-prod opened the pull request:
NL 

* Modifier le nom des boutons s’inscrire et se connecter en rentirant le "avec PIX" 
* Modifier ex par b.v.
* Modifier les placeholders par Jan Jansen
* Suppression du point sur Ik Ben Online (je me connecte)

ES :

* Modifier le nom des boutons s’inscrire et se connecter en retirant le "avec PIX"
* Modifier ex par e.j
* Modifier les placeholders par Juan Pérez

**https://app.recette.pix.org/connexion?lang=es**
* Registrarse en Pix --> "Regístrate en Pix" pour garder la cohérence du tutoiement.
* Me conecto --> "Iniciar sesión" .
* Otros medios de conexion --> "Otros métodos de conexión".

**https://app.recette.pix.org/inscription?lang=es**
* Acceder a Pix --> "Iniciar sesión" 
* Me inscribo --> "Crear cuenta".
* Otros medios de registro --> "Otros métodos de conexión".

**Composant d'aide à la création du mot de passe**

"Tu contraseña debe cumplir con los siguientes requisitos :
Mínimo 8 caracteres.
Al menos una letra mayúscula.
Al menos una letra minúscula.
Al menos un número."


**https://app.recette.pix.org/mot-de-passe-oublie?lang=es**
* Acceder a Pix --> "Iniciar sesión".
* Has olvidado --> "¿Olvidaste tu contraseña?".
* Recibir un enlace de reinicio --> "Reiniciar contraseña".
* No ha facilitado una dirección de correo electrónico --> "¿No ingresaste una dirección de correo electrónico?".

**Ecran final**
* Compruebe su correo electrónico --> "Comprueba tu correo electrónico".
* "Si tu dirección está asociada a una cuenta Pix, se te han enviado instrucciones por correo electrónico para restablecer tu contraseña."

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr10478.review.pix.fr)
  * [App (.org)](https://app-pr10478.review.pix.org)
  * [Orga (.fr)](https://orga-pr10478.review.pix.fr)
  * [Orga (.org)](https://orga-pr10478.review.pix.org)
  * [Certif (.fr)](https://certif-pr10478.review.pix.fr)
  * [Certif (.org)](https://certif-pr10478.review.pix.org)
  * [Junior](https://junior-pr10478.review.pix.fr)
  * [Admin](https://admin-pr10478.review.pix.fr)
  * [API](https://api-pr10478.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr10478.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr10478/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr10478/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr10478/environment)

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

AndreiaPena commented on mon-pix/translations/es.json:
```diff
@@ -66,9 +66,19 @@
+      "check": {
+        "detected": {
+          "description": "{version}<br />Extensión Pix Companion instalada y activada.",
+          "version": "{version}(versión )"
```
Ils se sont loupés sur cette clé, je corrige ça sur Phrase 

<img width="1010" alt="Capture d’écran 2024-11-05 à 17 52 57" src="https://github.com/user-attachments/assets/cbad3c14-9680-4e7c-bb5d-31f3384e74e5">
<img width="1010" alt="Capture d’écran 2024-11-05 à 17 53 01" src="https://github.com/user-attachments/assets/175f41ad-e426-44cf-a70f-c0f3a95807cb">

AndreiaPena replied:
C'est corrigé ! @MathGuich est-ce que c'est possible de pousser les modifs ? :)

MathGuich replied:
Je vais merger @AndreiaPena , ça te va ? j'ai vu que tu as poussé un truc via Phrase ce matin

AndreiaPena replied:
j'ai juste remis à jour sur dev, je n'ai rien poussé via Phrase. Je voulais te laisser la main dessus

MathGuich reviewed (approved):
Ok pour Accès

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state
